insights/terraform/projects/bcda/main.tf

```diff
@@ -60,6 +60,38 @@ module "glue_jobs" {
   ]
 }
 
+module "insights_data_table_dev" {
+  source      = "../../modules/table"
+  database    = module.database.name
+  table       = "dev_insights"
+  description = "dev insights data"
+  bucket      = module.bucket.id
+  bucket_cmk  = module.bucket.bucket_cmk
+  tags        = local.tags
+  partitions  = local.partitions
+  columns = [
```

Comment:
can the columns be made into a local and used in these tables to make the code more DRY?